code-challenges.js

```diff
@@ -29,17 +29,34 @@ const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
 // Pseudo code:
+// step 1: I want to write a code that will join the two arrays together using a + or a concat.
+// step 2: I want to find the length using the .length feature to count the total elements in the array.
+// step 3: write code console.log(padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length)
+// step 4: Output is 9
+console.log(padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length)
 
 // --------------------2) Create the code that will reverse the letters of a string using the test variable provided below.
 
-const currentCohort = "Golf 2023"
+//const currentCohort = "Golf 2023"
 // Expected output: "3202 floG"
 
 // Pseudo code:
+//step1: I want to make the string into an array using the .split("") function I used "" inside () to break up each word
+//step2: I used the .reverse() function to reverse each leatter
+//step3: I used .join() with "" to remove the semi-colons and return the array as a string.
+console.log(currentCohort.split("").reverse().join(""))
```

Comment:
- When you use .split(""), you said you are breaking up each word, but this will break up each character in the string. 
- `.reverse()` will reverse the array, not each letter.
- You said `.join` removes the semi-colons, but it will just convert the array into a string, combining each character since you put "" in the parentheses.